dev/com.ibm.ws.anno/publish/appData/secfvt_servlet30.ear/Servlet30DynPure.war/WEB-INF/web.xml

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
Some of these are a consequence of the testing, which took files from several sample applications as test data.

These tests should be refactored to pull in test data from other projects.

There is at least one case of java version dependent tests which has the .class files built outside of the usual build steps.  That avoids having to do compilations at different java versions in the same project, which would be complicated.

Reply:
Are you planning on resolving this as part of this PR?  Moving these tests to fats probably makes more sense.  If you are pulling in stuff from other fats you may want to do that as part of the fat instead of having the binary versions of the same classes checked into git a second time.

Reply:
I wasn't; I don't see that that can be done within the current delivery window.

Reply:
Do you have a follow up issue for this work?

Reply:
Being tracked for future cleanup in #9225.